CEP_8.x/FeatureList8.0

```diff
@@ -0,0 +1,27 @@
+Feature List
+==============
+* CEF and nodejs upgrade [Chromium version#57.0.2987.74 with nodeJS version#7.7.4]
+* IME Support on top of latest CEF
+* Windows High DPI Support – 125% scaling supported
+* Bug Fixes
+  * Crashes
+    * CEP extensions crashing in background
+    * Application hang in PlugPlugOWL (CEP) MonitorThreadProc while application quit
+    * MAC Only: CEPHtmlEngine crashes in background while using Illustrator.
+    * AI crashes during PlugPlugOWL is being unloaded.
```

Comment:
Use AI or Illustrator every where in the document.

Reply:
Done